hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1958,15 +1959,17 @@ private void addOMNodeToPeers(String newOMNodeId) throws IOException {
 
         if (newOMNodeDetails == null) {
           // If new node information is not present in the newly loaded
-          // configuration also, throw an exception
+          // configuration also, throw an exception.
+          // This case can also come when we have decommissioned a node and
+          // ratis will apply previous transactions to add that node back.
           throw new IOException("There is no OM configuration for node ID "
               + newOMNodeId + " in ozone-site.xml.");
```

Comment:
Instead of throwing `IOException` here and catching it in the below `catch` block, log an `error` message here and `return`.

Reply:
Done